pgrepo_test.go

```diff
@@ -1 +1,56 @@
-package tk_test
\ No newline at end of file
+package tk_test
+
+import (
+	"github.com/stretchr/testify/assert"
+	tokensync "github.com/tempcke/tk"
+	"os"
+	"testing"
+)
+
+func TestPgRepo(t *testing.T) {
+	var token tokensync.Token
+
+	err := os.Setenv(tokensync.DatabaseURL, "postgresql://actourex@localhost:5432/actourex")
+	assert.Nil(t, err)
+
+	pool, err := tokensync.GetPool(ctx)
+	assert.Nil(t, err)
+
+	repo, err := newFakePgRepo(pool)
+	assert.Nil(t, err)
+
+	defer repo.down()
+
+	t.Run("get token from pg repo storage when there are no tokens", func(t *testing.T) {
+		_, err = repo.GetToken(ctx)
+		assert.NotNil(t, err)
+	})
+
+	t.Run("store token into pg repo storage", func(t *testing.T) {
+		err = repo.StoreToken(ctx, newFakeToken())
+		assert.Nil(t, err)
```

Comment:
this test does not test anything not tested in the next test .... though I get there one step at a time and will have a test like this for a moment the very next step is to add a call to repo.GetToken and ensure I can read back the token correctly ... testing a store function without checking to see if you can get it back and that it is correct is not very useful by itself.

so that means just delete this test as it adds no value right now .. and I would rename the next test to "store and get token"